man/systemd.service.xml

```diff
@@ -113,6 +113,18 @@
     involved with early boot or late system shutdown should disable
     this option.</para>
 
+    <para>Note that instanced service units (i.e. service units with an <literal>@</literal> in their name) are by
+    default assigned a per-template slice unit (see
+    <citerefentry><refentrytitle>systemd.slice</refentrytitle><manvolnum>5</manvolnum></citerefentry>), named after the
+    template unit, containing all instances of the specific template. This slice is normally stopped at shutdown. Also
+    note that service units require their slice units, and hence if a slice unit is stopped all service units contained
+    in it are stopped, too. This means if you write a template service unit that shall stay around during late
+    shutdown, and is using <varname>DefaultDependencies=no</varname> to implement this, make sure to either define your
+    own per-template slice unit file that also sets <varname>DefaultDependencies=no</varname>, or to explicitly assign
+    your templated service unit to a different slice unit that already sets this, for example by using
+    <varname>Slice=system.slice</varname>. Also see
+    <citerefentry><refentrytitle>systemd.resource-control</refentrytitle><manvolnum>5</manvolnum></citerefentry>.</para>
```

Comment:
The explanation/solutions look correct (I tested both), but it's IMHO a bit too technical to understand. Maybe

This slice is normally stopped at shutdown, *together with all template instances. If that is not desired, set  <varname>DefaultDependencies=no</varname> in the template unit, and either* define your own per-template slice unit file that also sets <varname>DefaultDependencies=no</varname>, *or set <varname>Slice=system.slice</varname> (or another suitable slice) in the template unit.* Also see...